JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -190,16 +190,18 @@ module Microsoft.ApplicationInsights {
             if (!Telemetry.PageViewPerformance.isPerformanceTimingSupported()) {
                 // no navigation timing (IE 8, iOS Safari 8.4, Opera Mini 8 - see http://caniuse.com/#feat=nav-timing)
                 _InternalLogging.throwInternalNonUserActionable(LoggingSeverity.CRITICAL,
```

Comment:
I think at some stage everybody who cares about data quality will know that 5% of the data is unreliable due to the error in navigation API. If one needs more information - can customer detect this case and start reporting it himself? Maybe using telemetry initializer? Can we initialize the hidden field with the error descitption so it can be read by telemetry initializer and sent as custom property. My main concern is that we should not pollute telemetry with known errors.

For instance we do not report zero duration for requests telemetry but customer can increase error severity level and detect all such cases.